pubspec.yaml

```
@@ -5,4 +5,5 @@ environment:
   sdk: ^3.6.0
 
 workspace:
-  - apps
\ No newline at end of file
+  - apps
+  - packages/app_preferences
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)